web/src/test/kotlin/no/nav/su/se/bakover/web/external/FrikortRoutesKtTest.kt

```diff
@@ -57,7 +57,7 @@ internal class FrikortRoutesKtTest {
                 header(HttpHeaders.XCorrelationId, DEFAULT_CALL_ID)
                 header(
                     HttpHeaders.Authorization,
-                    jwtStub.createJwtToken(subject = applicationConfig.frikort.serviceUsername).asBearerToken(),
+                    jwtStub.createJwtToken(subject = applicationConfig.frikort.serviceUsername.first()).asBearerToken(),
```

Comment:
Kunne kanskje byttet en av de til second, eller kjørt testen i en loop 🤷

Reply:
For testene er det bare lagt inn en (fake) servicebruker, så second hadde krasja. Kunne lagd loop, men i det srvfrikorttjenester er oppe, så bør dette reverseres til å bare ta inn èn og ikke en liste igjen 😄